docs/references.bib

```diff
@@ -2101,6 +2101,14 @@ @Book{            kechris1995
   url           = {https://doi.org/10.1007/978-1-4612-4190-4}
 }
 
+@Book{            keng1982house,
+  author        = {Keng, Hua Loo},
+  publisher     = {Springer},
+  title         = {Introduction to Number Theory},
+  year          = {1982},
+  pages         = {489}
+}
```

Comment:
Unfortunately I think you have to put the reference in alphabetical order by hand, otherwise the bot gets crazy. If you want to complain feel free to write on Zulip :)